sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -32,18 +25,57 @@
               color="primary"
               data-cy="AcceptAssessment"
               @click="confirmAssessment()"
-              :disabled="warningList.length > 0"
+              :disabled="!canAcceptAssessment"
               >Accept assessment</v-btn
             >
           </v-row>
         </template>
       </header-navigator>
     </template>
+    <template #alerts>
+      <banner
+        class="mb-2"
+        header="Your assessment has warnings preventing it from being accepted"
+        :type="BannerTypes.Warning"
+        v-if="!canAcceptAssessment"
+      >
+        <template #content>
+          <ul>
+            <li v-if="eCertValidation.disabilityStatusNotConfirmed">
+              Your account has not been approved for disability funding. You
+              will not be able to accept this application until the account
+              disability status has been approved. If you would like to receive
+              all non disability funding please edit your application.
+            </li>
+            <li v-if="eCertValidation.msfaaInvalid">
+              Your MSFAA is not valid. Please complete your MSFAA with the
+              National Student Loans Centre to move forward with your
+              application. Please note, there is a one day delay between signing
+              your MSFAA and being able to accept your assessment.
+            </li>
+            <li v-if="eCertValidation.hasStopDisbursementRestriction">
+              You have restrictions that block funding on your account. Please
+              resolve them in order to move forward with your application.
+            </li>
+            <li v-if="eCertValidation.noEstimatedAwardAmounts">
```

Comment:
I have a question regarding the banner. 

What happens if the second disbursement has no estimated award?? is that possible?? and if so?? will this banner be displayed when the assessment is already accepted as first dirsbursement had some estimated awards?

Reply:
I will check.

Reply:
For the context of this PR, I had a chat with @CarlyCotton and applied the conditions to show the warnings following the below.

_e-Cert warnings will displayed when the conditions to accept the assessment are met and the button would be enabled if wasn't for the warnings displayed in the banner._

Reply:
Reply from the business with the most recent directions. https://github.com/bcgov/SIMS/issues/3382#issuecomment-2256567455

Reply:
Thanks for checking with business. Looks good to me.